### PR TITLE
Colors update

### PR DESCRIPTION
**Pull request by burakukula**
- Update of existing colors
 - Adding new special colors to the palet.

Before:

screen shot 2017-01-17 at 15 46 22
screen shot 2017-01-17 at 15 46 32
screen shot 2017-01-17 at 15 46 39

After:

screen shot 2017-01-17 at 15 47 09
screen shot 2017-01-17 at 15 47 16
screen shot 2017-01-17 at 15 47 22

### CODE REVIEW

**Comment by kdzwinel** [comment]
IMO we should not expose these colors to users. People should not use them, they are created for a specific usecase. I'm not even sure if we really want tournament/challenges icons in the SG.

**Comment by burakukula** [comment]
@kdzwinel I just leave them in config so we can use them in games. They are not visible in styleguide.

**Comment by kdzwinel** [comment]
Keeping only colors as variables should be :+1:

**Comment by burakukula** [comment]
@kdzwinel approved then? :)

File: src/sass/_config.scss [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
 // Grey colors
 $white: #ffffff;
-$grayPrimary: #70747d;
-$graySecondary: #a1a6b1;
-$grayLight: #dee2e5;
-$graySecondaryLight: #f0f3f5;
+$grayPrimary: #434e66;
+$graySecondary: #9fa6b5;
+$grayLight: #e1e8ed;
+$graySecondaryLightest: #f0f3f5;
```
Comment (kdzwinel) [inline src/sass/_config.scss]: graySecondaryLightest? why not graySecondaryLight?

Comment (burakukula) [inline src/sass/_config.scss]: @kdzwinel this is the first step to change the namespace:

grayLight -> graySecondaryLight
graySecondaryLight -> graySecondaryLightest

**Pull request merged**